libs/labelbox/src/labelbox/schema/project.py

```diff
@@ -1753,62 +1753,97 @@ def __check_data_rows_have_been_processed(
-            project(where: { id: $projectId }) {      
-            workstreamStateCounts {
-                state
-                count
-            }
-            taskQueues {
-                queueType
-                name
-                dataRowCount
-            }
-            issues {
-                totalCount
-            }
-            completedDataRowCount
+            query = """query ProjectGetOverviewPyApi($projectId: ID!) {
+                project(where: { id: $projectId }) {      
+                workstreamStateCounts {
+                    state
+                    count
+                }
+                taskQueues {
+                    queueType
+                    name
+                    dataRowCount
+                }
+                issues {
+                    totalCount
+                }
+                completedDataRowCount
+                }
             }
-        }
-        """
+            """
 
-        # Must use experimental to access "issues"
-        result = self.client.execute(query, {"projectId": self.uid},
-                                     experimental=True)["project"]
+            # Must use experimental to access "issues"
+            result = self.client.execute(query, {"projectId": self.uid},
+                                         experimental=True)["project"]
 
-        overview = {
-            utils.snake_case(st["state"]): st["count"]
-            for st in result.get("workstreamStateCounts")
-            if st["state"] != "NotInTaskQueue"
-        }
-
-        review_queues = {
-            tq["name"]: tq.get("dataRowCount")
-            for tq in result.get("taskQueues")
-            if tq.get("queueType") == "MANUAL_REVIEW_QUEUE"
-        }
-
-        # Store the total number of data rows in review
-        review_queues["all"] = overview.get("in_review")
-        overview["in_review"] = review_queues
+            # Reformat category names
+            overview = {
+                utils.snake_case(st["state"]): st["count"]
+                for st in result.get("workstreamStateCounts")
+                if st["state"] != "NotInTaskQueue"
+            }
 
-        overview["issues"] = result.get("issues", {}).get("totalCount")
+            overview["issues"] = result.get("issues", {}).get("totalCount")
 
-        # Rename keys
-        overview["to_label"] = overview.pop("unlabeled")
-        overview["all_in_data_rows"] = overview.pop("all")
+            # Rename categories
+            overview["to_label"] = overview.pop("unlabeled")
+            overview["total_data_rows"] = overview.pop("all")        
 
-        return ProjectOverview(**overview)
+            if not details:
+                return ProjectOverview(**overview)
+            else:
+                # Build queue details for review and rework queues
+                for category in ["rework", "review"]:
+                    overview[f"in_{category}"] = _build_queue_details(f"in_{category}", 
```

Comment:
is a nested function needed given how simple it is?

Reply:
Fixed